README.md

```diff
@@ -1 +1,37 @@
-# Grakn Example Projects and Tutorials
\ No newline at end of file
+# Grakn Examples
+
+This repository includes examples that showcase usage of Grakn Clients in reading from and writing to a Grakn knowledge graph.
+
+## Explore Client Examples
+
+Grakn officially supports development of clients for Java, Node.js and Python. Learn more about [Grakn Clients](http://dev.grakn.ai/docs/client-api/overview).
```

Comment:
Let's do this:
_Grakn officially supports ~~development of~~ clients for Java, Node.js and Python._
the fact they are continually developed is implied from the fact that they're officially supported